xml/ns-System.Text.Json.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="System.Text.Json">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>The System.Text.Json namespace provides high-performance, low-allocating, and RFC-compliant capabilities to process JavaScript Object Notation (JSON), which includes serializing objects to JSON text and deserializing JSON text to objects, with UTF-8 support built-in. It also provides types to read and write JSON text encoded as UTF-8, and to create a read-only in-memory document object model (DOM) for random access of the JSON elements within a structured view of the data.</summary>
```

Comment:
```suggestion
    <summary>The System.Text.Json namespace provides high-performance, low-allocating, and RFC-compliant capabilities to process JavaScript Object Notation (JSON), which includes serializing objects to JSON text and deserializing JSON text to objects, with UTF-8 support built-in. It also provides types to read and write JSON text encoded as UTF-8, and to create an in-memory document object model (DOM) for random access of the JSON elements within a structured view of the data.</summary>
```

Reply:
Re-reading this, I am not sure if we should say "RFC-compliant" or standards-compliant (similar to XML).

"json standard" gets a lot more hits when searching than "json rfc" although we are following RFC 8529: https://tools.ietf.org/html/rfc8259

Reply:
Discussed this a bit more with @GrabYourPitchforks and he brought up a good point about using a term like standard that is more generic/wider reach (and friendlier for reading in different languages).

Let's switch to that instead.

```suggestion
    <summary>The System.Text.Json namespace provides high-performance, low-allocating, and standards-compliant capabilities to process JavaScript Object Notation (JSON), which includes serializing objects to JSON text and deserializing JSON text to objects, with UTF-8 support built-in. It also provides types to read and write JSON text encoded as UTF-8, and to create an in-memory document object model (DOM) for random access of the JSON elements within a structured view of the data.</summary>
```

Reply:
Thanks for the suggestions. I'll apply it.